lib/tasks/ci.rake

```diff
@@ -2,7 +2,11 @@ ZIP_FILE = 'https://github.com/projecthydra/hydra-jetty/archive/v7.3.1.zip'
 
 desc "Run Continuous Integration Suite (tests, coverage, docs)"
 task :ci => ['jetty:clean', 'jetty:config'] do
-  Rails.env = "test"
+  unless Rails.env.test?
+    system('RAILS_ENV=test rake ci')
```

Comment:
why is this needed? Won't this cause `'jetty:clean', 'jetty:config'` to run twice?